internal/utils/utils.go

```diff
@@ -0,0 +1,36 @@
+package utils
```

Comment:
Since this is a helper for working with schemas, I think we could have `package schema` and this file could be `set_map.go`